pkg/scalers/rabbitmq_scaler.go

```diff
@@ -26,9 +33,17 @@ type rabbitMQScaler struct {
 }
 
 type rabbitMQMetadata struct {
-	queueName   string
-	host        string
-	queueLength int
+	queueName    string
+	host         string // connection string for AMQP protocol
+	apiHost      string // connection string for management API requests
+	queueLength  int
+	countUnacked bool // if true uses HTTP API and requires apiHost, if false uses AMQP and requires host
```

Comment:
What about `includeUnacked`? Now it feels like it will only include unacked messages

Reply:
that makes sense